Sources/SnapshottingTests/PreviewLayoutTest.swift

```diff
@@ -10,25 +10,46 @@ import Foundation
 import SwiftUI
 import XCTest
 
-// Test Xcode previews by forcing a layout pass of each one
+/// A test class for verifying Xcode previews by forcing a layout pass on each one.
+///
+/// This class is designed to discover and test SwiftUI previews to ensure they can be laid out without crashing.
+/// It provides mechanisms for filtering previews and performs layout tests on different platforms.
 open class PreviewLayoutTest: PreviewBaseTest, PreviewFilters {
 
+  /// Returns an optional array of preview names to be included in the snapshot testing. This also supports Regex format.
```

Comment:
I was hoping these methods could be documented in the `PreviewFilters` interface so it doesn't have to be duplicated in this file and SnapshotTest.swift, is that possible or is that not how docc works?